database/migrations/postgres/1732637175997202_releases_autoincrement.up.sql

```diff
@@ -1,4 +1,12 @@
-ALTER TABLE IF EXISTS releases ADD COLUMN IF NOT EXISTS version INTEGER;
+DO $$ 
```

Comment:
Why are we changing an old migration? We should generally not do that, except if it's a bug fix.

Reply:
it's actually a bug fix. The issue is that it wasn't idempotent. So, after the new migrations are applied and then we rerun the migrations we get an error. The integration tests were failing because of that.
But, it wouldn't be a problem if these migrations are already applied somewhere.